### PR TITLE
SDN-4013: Add `OVNCrashOnDualStackMigrated` for 4.12.{1,2,3,4}

### PR DESCRIPTION
**Pull request by petr-muller**

### CODE REVIEW

File: blocked-edges/4.12.1-OVNCrashOnDualStackMigrated.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+to: 4.12.1
+from: .*
+url: https://issues.redhat.com/browse/SDN-4013
+name: OVNCrashOnDualStackMigrated
```
Comment (LalatenduMohanty) [inline blocked-edges/4.12.1-OVNCrashOnDualStackMigrated.yaml:4]: Can we change it to `OVNCrashOnMigratedDualStack` ?

**Review by LalatenduMohanty — CHANGES_REQUESTED** [review]

File: blocked-edges/4.12.1-OVNCrashOnDualStackMigrated.yaml (hunk 1 above)
Comment (petr-muller) [reply blocked-edges/4.12.1-OVNCrashOnDualStackMigrated.yaml:4]: Sure

**Review by LalatenduMohanty — APPROVED** [review]
/lgtm

**Comment by wking** [comment]
[Quay 502][1].

/retest

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cincinnati-graph-data/3776/pull-ci-openshift-cincinnati-graph-data-master-e2e/1671876885023297536

**Pull request merged**